cli/commands/update.go

```diff
@@ -97,6 +98,7 @@ func parseUpdateResponse(responseBytes []byte) (string, error) {
 }
 
 func doUpdate(optionsFile, packageVersion string) {
+	config.Command = "update"
```

Comment:
This seems like it has weird side effects. Config is thrown away after the command completes?